recipe/build.sh

```diff
@@ -29,7 +29,12 @@ popd
 if [[ "$CONDA_BUILD_CROSS_COMPILATION" == "1" ]]; then
   # llvm-config from host env is tried by cctools-port
   # Move the build prefix one to host prefix
-  mv $BUILD_PREFIX/bin/llvm-config $PREFIX/bin/llvm-config
+  rm $PREFIX/bin/llvm-config
```

Comment:
I know that it shouldn't be normally necessary, but I think specifying here a '-f' is more sane, as even if that file doesn't exist at that place, we want to move it in from other place, isn't it?
Otherwise changes are looking good to me.

Reply:
Yes, I'll clean it up. The INSTALL_NAME_TOOL hack is also not needed because `llvm-config` no longer links with `libLLVM.dylib` because of https://github.com/conda-forge/llvmdev-feedstock/commit/07506fbe731f9e716a4892d16898fba4acd40062